.github/workflows/verify-go.yml

```diff
@@ -64,7 +64,7 @@ jobs:
     - name: checkout
       uses: actions/checkout@v2
     - name: build
-      run: CGO_ENABLED=0 go build -o piper && chmod +x piper
+      run: CGO_ENABLED=0 go build -o piper -tags release && chmod +x piper
```

Comment:
I would like to suggest to add some docu here with a comment why we have that `--tags release`. Otherwise somebody after not knowing the details might decide to remove this (e.g. since the build succeeds also without that flag).